src/components/Navbar/components/SearchBar/index.css

```diff
@@ -8,7 +8,7 @@
 .searchInput {
   width: 388px;
   height: 39px;
-  background: #1a1b1c;
+  background: rgba(26, 27, 28, 0.5);
```

Comment:
Can you create a variable for that? Since we are planning on making Heroic themes in the future, we want to use variables for all colors.